Rename all contentextensions *.*.json files to *.json

fujii opened the pull request:
#### 8bb2ce6b3ecc587858c384dfce6e2bc5d4a3434f
<pre>
Rename all contentextensions *.*.json files to *.json
<a href="https://bugs.webkit.org/show_bug.cgi?id=304302">https://bugs.webkit.org/show_bug.cgi?id=304302</a>

Reviewed by Alexey Proskuryakov.

*.py.json files were treated as Python CGI because they contained .py in the
file names. &lt;<a href="https://commits.webkit.org/304505@main">https://commits.webkit.org/304505@main</a>&gt; fixed the problem with a
Apache config hack. We had a discussion about a better solution after the fix.

Reverted the Apache config part of 304505@main. And, renamed all *.*.json to
*.json not to contain .py in the file names. Changed two
TestController::configureContentExtensionForTest implementations to use the new
json file names.

* LayoutTests/http/conf/apache2.2-httpd.conf:
* LayoutTests/http/conf/apache2.4-darwin-httpd.conf:
* LayoutTests/http/conf/apache2.4-httpd.conf:
* LayoutTests/http/conf/archlinux-httpd.conf:
* LayoutTests/http/conf/debian-httpd-2.4.conf:
* LayoutTests/http/conf/fedora-httpd-2.2.conf:
* LayoutTests/http/conf/fedora-httpd-2.4.conf:
* LayoutTests/http/conf/win-httpd-2.4.conf:
* LayoutTests/http/tests/contentextensions/allowlist.json: Renamed from LayoutTests/http/tests/contentextensions/allowlist.html.json.
* LayoutTests/http/tests/contentextensions/async-xhr-onerror.json: Renamed from LayoutTests/http/tests/contentextensions/async-xhr-onerror.html.json.
* LayoutTests/http/tests/contentextensions/basic-filter.json: Renamed from LayoutTests/http/tests/contentextensions/basic-filter.html.json.
* LayoutTests/http/tests/contentextensions/block-child-document-resource-type.json: Renamed from LayoutTests/http/tests/contentextensions/block-child-document-resource-type.html.json.
* LayoutTests/http/tests/contentextensions/block-cookies-basic.json: Renamed from LayoutTests/http/tests/contentextensions/block-cookies-basic.html.json.
* LayoutTests/http/tests/contentextensions/block-cookies-in-csp-report.json: Renamed from LayoutTests/http/tests/contentextensions/block-cookies-in-csp-report.py.json.
* LayoutTests/http/tests/contentextensions/block-cookies-in-image-load-in-onunload.json: Renamed from LayoutTests/http/tests/contentextensions/block-cookies-in-image-load-in-onunload.html.json.
* LayoutTests/http/tests/contentextensions/block-cookies-in-ping.json: Renamed from LayoutTests/http/tests/contentextensions/block-cookies-in-ping.html.json.
* LayoutTests/http/tests/contentextensions/block-cookies-send.https.json: Renamed from LayoutTests/http/tests/contentextensions/block-cookies-send.https.html.json.
* LayoutTests/http/tests/contentextensions/block-csp-report.json: Renamed from LayoutTests/http/tests/contentextensions/block-csp-report.py.json.
* LayoutTests/http/tests/contentextensions/block-everything-if-domain.json: Renamed from LayoutTests/http/tests/contentextensions/block-everything-if-domain.html.json.
* LayoutTests/http/tests/contentextensions/block-everything-unless-domain-iframe.json: Renamed from LayoutTests/http/tests/contentextensions/block-everything-unless-domain-iframe.html.json.
* LayoutTests/http/tests/contentextensions/block-everything-unless-domain-redirect.json: Renamed from LayoutTests/http/tests/contentextensions/block-everything-unless-domain-redirect.py.json.
* LayoutTests/http/tests/contentextensions/block-everything-unless-domain.json: Renamed from LayoutTests/http/tests/contentextensions/block-everything-unless-domain.html.json.
* LayoutTests/http/tests/contentextensions/block-image-load-in-onunload.json: Renamed from LayoutTests/http/tests/contentextensions/block-image-load-in-onunload.html.json.
* LayoutTests/http/tests/contentextensions/block-ping-resource-type-ping.json: Renamed from LayoutTests/http/tests/contentextensions/block-ping-resource-type-ping.html.json.
* LayoutTests/http/tests/contentextensions/block-ping-resource-type-raw.json: Renamed from LayoutTests/http/tests/contentextensions/block-ping-resource-type-raw.html.json.
* LayoutTests/http/tests/contentextensions/block-ping.json: Renamed from LayoutTests/http/tests/contentextensions/block-ping.html.json.
* LayoutTests/http/tests/contentextensions/block-private-click-measurement.json: Renamed from LayoutTests/http/tests/contentextensions/block-private-click-measurement.html.json.
* LayoutTests/http/tests/contentextensions/block-top-document-resource-type-with-child-document.json: Renamed from LayoutTests/http/tests/contentextensions/block-top-document-resource-type-with-child-document.html.json.
* LayoutTests/http/tests/contentextensions/block-top-document-resource-type.json: Renamed from LayoutTests/http/tests/contentextensions/block-top-document-resource-type.html.json.
* LayoutTests/http/tests/contentextensions/character-set-basic-support.json: Renamed from LayoutTests/http/tests/contentextensions/character-set-basic-support.html.json.
* LayoutTests/http/tests/contentextensions/crash-xhr.json: Renamed from LayoutTests/http/tests/contentextensions/crash-xhr.html.json.
* LayoutTests/http/tests/contentextensions/cross-origin-content-rules.json: Renamed from LayoutTests/http/tests/contentextensions/cross-origin-content-rules.html.json.
* LayoutTests/http/tests/contentextensions/css-display-none-after-ignore-previous-rules.json: Renamed from LayoutTests/http/tests/contentextensions/css-display-none-after-ignore-previous-rules.html.json.
* LayoutTests/http/tests/contentextensions/css-display-none-font-shadow-tree.json: Renamed from LayoutTests/http/tests/contentextensions/css-display-none-font-shadow-tree.html.json.
* LayoutTests/http/tests/contentextensions/css-display-none-keyframe.json: Renamed from LayoutTests/http/tests/contentextensions/css-display-none-keyframe.html.json.
* LayoutTests/http/tests/contentextensions/css-display-none-overflows-rule-data-1.json: Renamed from LayoutTests/http/tests/contentextensions/css-display-none-overflows-rule-data-1.html.json.
* LayoutTests/http/tests/contentextensions/css-display-none-overflows-rule-data-2.json: Renamed from LayoutTests/http/tests/contentextensions/css-display-none-overflows-rule-data-2.html.json.
* LayoutTests/http/tests/contentextensions/css-display-none-overflows-rule-data-3.json: Renamed from LayoutTests/http/tests/contentextensions/css-display-none-overflows-rule-data-3.html.json.
* LayoutTests/http/tests/contentextensions/css-display-none-overflows-rule-data-4.json: Renamed from LayoutTests/http/tests/contentextensions/css-display-none-overflows-rule-data-4.html.json.
* LayoutTests/http/tests/contentextensions/css-display-none-overflows-rule-data-5.json: Renamed from LayoutTests/http/tests/contentextensions/css-display-none-overflows-rule-data-5.html.json.
* LayoutTests/http/tests/contentextensions/css-display-none-overflows-rule-data-6.json: Renamed from LayoutTests/http/tests/contentextensions/css-display-none-overflows-rule-data-6.html.json.
* LayoutTests/http/tests/contentextensions/css-display-none-with-different-case-sensitivity-are-not-merged.json: Renamed from LayoutTests/http/tests/contentextensions/css-display-none-with-different-case-sensitivity-are-not-merged.html.json.
* LayoutTests/http/tests/contentextensions/css-display-none-with-different-triggers-are-not-merged.json: Renamed from LayoutTests/http/tests/contentextensions/css-display-none-with-different-triggers-are-not-merged.html.json.
* LayoutTests/http/tests/contentextensions/css-display-none.json: Renamed from LayoutTests/http/tests/contentextensions/css-display-none.html.json.
* LayoutTests/http/tests/contentextensions/dns-prefetch-blocked.json: Renamed from LayoutTests/http/tests/contentextensions/dns-prefetch-blocked.html.json.
* LayoutTests/http/tests/contentextensions/domain-rules.json: Renamed from LayoutTests/http/tests/contentextensions/domain-rules.html.json.
* LayoutTests/http/tests/contentextensions/fetch-redirect-blocked.json: Renamed from LayoutTests/http/tests/contentextensions/fetch-redirect-blocked.html.json.
* LayoutTests/http/tests/contentextensions/filters-with-quantifiers-combined.json: Renamed from LayoutTests/http/tests/contentextensions/filters-with-quantifiers-combined.html.json.
* LayoutTests/http/tests/contentextensions/font-display-none-repeated-layout.json: Renamed from LayoutTests/http/tests/contentextensions/font-display-none-repeated-layout.html.json.
* LayoutTests/http/tests/contentextensions/frame-url.json: Renamed from LayoutTests/http/tests/contentextensions/frame-url.html.json.
* LayoutTests/http/tests/contentextensions/hide-on-csp-report.json: Renamed from LayoutTests/http/tests/contentextensions/hide-on-csp-report.py.json.
* LayoutTests/http/tests/contentextensions/hide-on-ping-with-ping-that-redirects.json: Renamed from LayoutTests/http/tests/contentextensions/hide-on-ping-with-ping-that-redirects.html.json.
* LayoutTests/http/tests/contentextensions/hide-on-ping.json: Renamed from LayoutTests/http/tests/contentextensions/hide-on-ping.html.json.
* LayoutTests/http/tests/contentextensions/invalid-selector.json: Renamed from LayoutTests/http/tests/contentextensions/invalid-selector.html.json.
* LayoutTests/http/tests/contentextensions/main-resource-redirect-blocked.json: Renamed from LayoutTests/http/tests/contentextensions/main-resource-redirect-blocked.py.json.
* LayoutTests/http/tests/contentextensions/main-resource-redirect-error.json: Renamed from LayoutTests/http/tests/contentextensions/main-resource-redirect-error.html.json.
* LayoutTests/http/tests/contentextensions/main-resource.json: Renamed from LayoutTests/http/tests/contentextensions/main-resource.html.json.
* LayoutTests/http/tests/contentextensions/make-https.json: Renamed from LayoutTests/http/tests/contentextensions/make-https.html.json.
* LayoutTests/http/tests/contentextensions/media-filtered.json: Renamed from LayoutTests/http/tests/contentextensions/media-filtered.html.json.
* LayoutTests/http/tests/contentextensions/popups.json: Renamed from LayoutTests/http/tests/contentextensions/popups.html.json.
* LayoutTests/http/tests/contentextensions/preconnect-blocked.json: Renamed from LayoutTests/http/tests/contentextensions/preconnect-blocked.html.json.
* LayoutTests/http/tests/contentextensions/prefetch-blocked.json: Renamed from LayoutTests/http/tests/contentextensions/prefetch-blocked.html.json.
* LayoutTests/http/tests/contentextensions/reload-without-contentextensions.json: Renamed from LayoutTests/http/tests/contentextensions/reload-without-contentextensions.html.json.
* LayoutTests/http/tests/contentextensions/root-actions.json: Renamed from LayoutTests/http/tests/contentextensions/root-actions.html.json.
* LayoutTests/http/tests/contentextensions/script-onerror.json: Renamed from LayoutTests/http/tests/contentextensions/script-onerror.html.json.
* LayoutTests/http/tests/contentextensions/service-worker-block-everything-if-localhost.json: Renamed from LayoutTests/http/tests/contentextensions/service-worker-block-everything-if-localhost.html.json.
* LayoutTests/http/tests/contentextensions/service-worker.https.json: Renamed from LayoutTests/http/tests/contentextensions/service-worker.https.html.json.
* LayoutTests/http/tests/contentextensions/single-css-display-none.json: Renamed from LayoutTests/http/tests/contentextensions/single-css-display-none.html.json.
* LayoutTests/http/tests/contentextensions/style-resolver-changed-reentrancy.json: Renamed from LayoutTests/http/tests/contentextensions/style-resolver-changed-reentrancy.html.json.
* LayoutTests/http/tests/contentextensions/subresource-redirect-blocked.json: Renamed from LayoutTests/http/tests/contentextensions/subresource-redirect-blocked.html.json.
* LayoutTests/http/tests/contentextensions/sync-xhr-blocked.json: Renamed from LayoutTests/http/tests/contentextensions/sync-xhr-blocked.html.json.
* LayoutTests/http/tests/contentextensions/sync-xhr-redirection-blocked.json: Renamed from LayoutTests/http/tests/contentextensions/sync-xhr-redirection-blocked.html.json.
* LayoutTests/http/tests/contentextensions/test-jump-table-bytecode-generation.json: Renamed from LayoutTests/http/tests/contentextensions/test-jump-table-bytecode-generation.html.json.
* LayoutTests/http/tests/contentextensions/text-track-blocked.json: Renamed from LayoutTests/http/tests/contentextensions/text-track-blocked.html.json.
* LayoutTests/http/tests/contentextensions/top-url.json: Renamed from LayoutTests/http/tests/contentextensions/top-url.html.json.
* LayoutTests/http/tests/contentextensions/two-distinguishable-css-display-none-rules-on-main-resource.json: Renamed from LayoutTests/http/tests/contentextensions/two-distinguishable-css-display-none-rules-on-main-resource.html.json.
* LayoutTests/http/tests/contentextensions/video-element-resource-type.json: Renamed from LayoutTests/http/tests/contentextensions/video-element-resource-type.html.json.
* LayoutTests/http/tests/inspector/network/contentextensions/blocked-websocket-crash.json: Renamed from LayoutTests/http/tests/inspector/network/contentextensions/blocked-websocket-crash.html.json.
* LayoutTests/http/tests/websocket/tests/hybi/contentextensions/block-cookies-worker.json: Copied from LayoutTests/http/tests/websocket/tests/hybi/contentextensions/block.html.json.
* LayoutTests/http/tests/websocket/tests/hybi/contentextensions/block-cookies-worker.py.json: Removed.
* LayoutTests/http/tests/websocket/tests/hybi/contentextensions/block-cookies.json: Copied from LayoutTests/http/tests/websocket/tests/hybi/contentextensions/block.html.json.
* LayoutTests/http/tests/websocket/tests/hybi/contentextensions/block-cookies.py.json: Removed.
* LayoutTests/http/tests/websocket/tests/hybi/contentextensions/block-worker.json: Renamed from LayoutTests/http/tests/websocket/tests/hybi/contentextensions/block-worker.html.json.
* LayoutTests/http/tests/websocket/tests/hybi/contentextensions/block.json: Renamed from LayoutTests/http/tests/websocket/tests/hybi/contentextensions/block.html.json.
* LayoutTests/http/tests/websocket/tests/hybi/contentextensions/display-none-worker.json: Renamed from LayoutTests/http/tests/websocket/tests/hybi/contentextensions/display-none-worker.html.json.
* LayoutTests/http/tests/websocket/tests/hybi/contentextensions/display-none.json: Renamed from LayoutTests/http/tests/websocket/tests/hybi/contentextensions/display-none.html.json.
* LayoutTests/http/tests/websocket/tests/hybi/contentextensions/upgrade-worker.json: Renamed from LayoutTests/http/tests/websocket/tests/hybi/contentextensions/upgrade-worker.html.json.
* LayoutTests/http/tests/websocket/tests/hybi/contentextensions/upgrade.json: Renamed from LayoutTests/http/tests/websocket/tests/hybi/contentextensions/upgrade.html.json.
* LayoutTests/http/wpt/beacon/contentextensions/beacon-blocked.json: Renamed from LayoutTests/http/wpt/beacon/contentextensions/beacon-blocked.html.json.
* LayoutTests/http/wpt/beacon/contentextensions/beacon-redirect-blocked.json: Renamed from LayoutTests/http/wpt/beacon/contentextensions/beacon-redirect-blocked.html.json.
* LayoutTests/platform/glib/TestExpectations:
* Tools/WebKitTestRunner/TestController.cpp:
* Tools/WebKitTestRunner/mac/TestControllerMac.mm:
(WTR::TestController::configureContentExtensionForTest):

Canonical link: <a href="https://commits.webkit.org/304625@main">https://commits.webkit.org/304625@main</a>
</pre>
<!--EWS-Status-Bubble-Start-->
https://github.com/WebKit/WebKit/commit/4d750fefcc9a68f619e87755dc13624e0ba8609b

| Misc | iOS, visionOS, tvOS & watchOS  | macOS  | Linux |  Windows |
| ----- | ---------------------- | ------- |  ----- |  --------- |
| [✅ 🧪 style](https://ews-build.webkit.org/#/builders/38/builds/136030 "Passed style check") | [✅ 🛠 ios](https://ews-build.webkit.org/#/builders/159/builds/8382 "Built successfully") | [✅ 🛠 mac](https://ews-build.webkit.org/#/builders/138/builds/47304 "Built successfully") | [✅ 🛠 wpe](https://ews-build.webkit.org/#/builders/5/builds/143729 "Built successfully") | [  ~~🛠 win~~](https://ews-build.webkit.org/#/builders/59/builds/88999 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") 
| [✅ 🧪 bindings](https://ews-build.webkit.org/#/builders/9/builds/137899 "Passed tests") | [✅ 🛠 ios-sim](https://ews-build.webkit.org/#/builders/155/builds/9051 "Built successfully") | [✅ 🛠 mac-AS-debug](https://ews-build.webkit.org/#/builders/156/builds/8229 "Built successfully") | [✅ 🧪 wpe-wk2](https://ews-build.webkit.org/#/builders/34/builds/103951 "Passed tests") | [  ~~🧪 win-tests~~](https://ews-build.webkit.org/#/builders/59/builds/88999 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") 
| [✅ 🧪 webkitperl](https://ews-build.webkit.org/#/builders/11/builds/138976 "Passed tests") | [✅ 🧪 ios-wk2](https://ews-build.webkit.org/#/builders/162/builds/6564 "Passed tests") | [⏳ 🧪 api-mac](https://ews-build.webkit.org/#/builders/API-Tests-macOS-EWS "Waiting to run tests") | [❌ 🧪 api-wpe](https://ews-build.webkit.org/#/builders/41/builds/84830 "Found 1 new API test failure: WPE/TestWebKitWebView:/webkit/WebKitWebView/save (failure)") | 
| | [✅ 🧪 ios-wk2-wpt](https://ews-build.webkit.org/#/builders/154/builds/6254 "Passed tests") | [✅ 🧪 api-mac-debug](https://ews-build.webkit.org/#/builders/165/builds/3898 "Passed tests") | [✅ 🛠 wpe-cairo-libwebrtc](https://ews-build.webkit.org/#/builders/166/builds/4330 "Built successfully") | 
| | [✅ 🧪 api-ios](https://ews-build.webkit.org/#/builders/13/builds/115523 "Passed tests") | [✅ 🧪 mac-wk1](https://ews-build.webkit.org/#/builders/135/builds/40089 "Passed tests") | [✅ 🛠 gtk](https://ews-build.webkit.org/#/builders/2/builds/146482 "Built successfully") | 
| | [✅ 🛠 vision](https://ews-build.webkit.org/#/builders/153/builds/8065 "Built successfully") | [✅ 🧪 mac-wk2](https://ews-build.webkit.org/#/builders/136/builds/40656 "Passed tests") | [✅ 🧪 gtk-wk2](https://ews-build.webkit.org/#/builders/1/builds/112306 "Passed tests") | 
| | [✅ 🛠 vision-sim](https://ews-build.webkit.org/#/builders/160/builds/8081 "Built successfully") | [✅ 🧪 mac-AS-debug-wk2](https://ews-build.webkit.org/#/builders/161/builds/6767 "Passed tests") | [❌ 🧪 api-gtk](https://ews-build.webkit.org/#/builders/21/builds/112705 "Found 36 new API test failures: WebKitGTK/TestWebKitAccessibility:/webkit/WebKitAccessibility/text/iterator, WebKitGTK/TestWebKitAccessibility:/webkit/WebKitAccessibility/accessible/state, WebKitGTK/TestInspectorServer:/webkit/WebKitWebInspectorServer/test-page-list, WebKitGTK/TestWebKitAccessibility:/webkit/WebKitAccessibility/document/basic, WebKitGTK/TestWebKitAccessibility:/webkit/WebKitAccessibility/image/basic, WebKitGTK/TestWebKitWebView:/webkit/WebKitWebView/submit-form, WebKitGTK/TestWebKitAccessibility:/webkit/WebKitAccessibility/hyperlink/basic, WebKitGTK/TestWebKitAccessibility:/webkit/WebKitAccessibility/text/attributes, WebKitGTK/TestWebKitAccessibility:/webkit/WebKitAccessibility/collection/get-matches, WebKitGTK/TestWebKitAccessibility:/webkit/WebKitAccessibility/text/state-changed ... (failure)") | 
| [✅ 🛠 🧪 merge](https://ews-build.webkit.org/#/builders/19/builds/28616 "Built successfully and passed tests") | [✅ 🧪 vision-wk2](https://ews-build.webkit.org/#/builders/164/builds/6154 "Passed tests") | [✅ 🧪 mac-wk2-stress](https://ews-build.webkit.org/#/builders/8/builds/118198 "Passed tests") | [  ~~🛠 playstation~~](https://ews-build.webkit.org/#/builders/134/builds/62041 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | 
| | [✅ 🛠 tv](https://ews-build.webkit.org/#/builders/158/builds/8115 "Built successfully") | [⏳ 🧪 mac-intel-wk2](https://ews-build.webkit.org/#/builders/macOS-Sonoma-Release-WK2-Intel-Tests-EWS "Waiting to run tests") | | 
| | [✅ 🛠 tv-sim](https://ews-build.webkit.org/#/builders/157/builds/7831 "Built successfully") | | | 
| | [✅ 🛠 watch](https://ews-build.webkit.org/#/builders/163/builds/8052 "Built successfully") | | | 
| | [✅ 🛠 watch-sim](https://ews-build.webkit.org/#/builders/152/builds/7911 "Built successfully") | | | 
<!--EWS-Status-Bubble-End-->

aproskuryakov reviewed (approved):
Thank you for going above and beyond!

fujii committed:
Rename all contentextensions *.*.json files to *.json
https://bugs.webkit.org/show_bug.cgi?id=304302

Reviewed by Alexey Proskuryakov.

*.py.json files were treated as Python CGI because they contained .py in the
file names. <https://commits.webkit.org/304505@main> fixed the problem with a
Apache config hack. We had a discussion about a better solution after the fix.

Reverted the Apache config part of 304505@main. And, renamed all *.*.json to
*.json not to contain .py in the file names. Changed two
TestController::configureContentExtensionForTest implementations to use the new
json file names.

* LayoutTests/http/conf/apache2.2-httpd.conf:
* LayoutTests/http/conf/apache2.4-darwin-httpd.conf:
* LayoutTests/http/conf/apache2.4-httpd.conf:
* LayoutTests/http/conf/archlinux-httpd.conf:
* LayoutTests/http/conf/debian-httpd-2.4.conf:
* LayoutTests/http/conf/fedora-httpd-2.2.conf:
* LayoutTests/http/conf/fedora-httpd-2.4.conf:
* LayoutTests/http/conf/win-httpd-2.4.conf:
* LayoutTests/http/tests/contentextensions/allowlist.json: Renamed from LayoutTests/http/tests/contentextensions/allowlist.html.json.
* LayoutTests/http/tests/contentextensions/async-xhr-onerror.json: Renamed from LayoutTests/http/tests/contentextensions/async-xhr-onerror.html.json.
* LayoutTests/http/tests/contentextensions/basic-filter.json: Renamed from LayoutTests/http/tests/contentextensions/basic-filter.html.json.
* LayoutTests/http/tests/contentextensions/block-child-document-resource-type.json: Renamed from LayoutTests/http/tests/contentextensions/block-child-document-resource-type.html.json.
* LayoutTests/http/tests/contentextensions/block-cookies-basic.json: Renamed from LayoutTests/http/tests/contentextensions/block-cookies-basic.html.json.
* LayoutTests/http/tests/contentextensions/block-cookies-in-csp-report.json: Renamed from LayoutTests/http/tests/contentextensions/block-cookies-in-csp-report.py.json.
* LayoutTests/http/tests/contentextensions/block-cookies-in-image-load-in-onunload.json: Renamed from LayoutTests/http/tests/contentextensions/block-cookies-in-image-load-in-onunload.html.json.
* LayoutTests/http/tests/contentextensions/block-cookies-in-ping.json: Renamed from LayoutTests/http/tests/contentextensions/block-cookies-in-ping.html.json.
* LayoutTests/http/tests/contentextensions/block-cookies-send.https.json: Renamed from LayoutTests/http/tests/contentextensions/block-cookies-send.https.html.json.
* LayoutTests/http/tests/contentextensions/block-csp-report.json: Renamed from LayoutTests/http/tests/contentextensions/block-csp-report.py.json.
* LayoutTests/http/tests/contentextensions/block-everything-if-domain.json: Renamed from LayoutTests/http/tests/contentextensions/block-everything-if-domain.html.json.
* LayoutTests/http/tests/contentextensions/block-everything-unless-domain-iframe.json: Renamed from LayoutTests/http/tests/contentextensions/block-everything-unless-domain-iframe.html.json.
* LayoutTests/http/tests/contentextensions/block-everything-unless-domain-redirect.json: Renamed from LayoutTests/http/tests/contentextensions/block-everything-unless-domain-redirect.py.json.
* LayoutTests/http/tests/contentextensions/block-everything-unless-domain.json: Renamed from LayoutTests/http/tests/contentextensions/block-everything-unless-domain.html.json.
* LayoutTests/http/tests/contentextensions/block-image-load-in-onunload.json: Renamed from LayoutTests/http/tests/contentextensions/block-image-load-in-onunload.html.json.
* LayoutTests/http/tests/contentextensions/block-ping-resource-type-ping.json: Renamed from LayoutTests/http/tests/contentextensions/block-ping-resource-type-ping.html.json.
* LayoutTests/http/tests/contentextensions/block-ping-resource-type-raw.json: Renamed from LayoutTests/http/tests/contentextensions/block-ping-resource-type-raw.html.json.
* LayoutTests/http/tests/contentextensions/block-ping.json: Renamed from LayoutTests/http/tests/contentextensions/block-ping.html.json.
* LayoutTests/http/tests/contentextensions/block-private-click-measurement.json: Renamed from LayoutTests/http/tests/contentextensions/block-private-click-measurement.html.json.
* LayoutTests/http/tests/contentextensions/block-top-document-resource-type-with-child-document.json: Renamed from LayoutTests/http/tests/contentextensions/block-top-document-resource-type-with-child-document.html.json.
* LayoutTests/http/tests/contentextensions/block-top-document-resource-type.json: Renamed from LayoutTests/http/tests/contentextensions/block-top-document-resource-type.html.json.
* LayoutTests/http/tests/contentextensions/character-set-basic-support.json: Renamed from LayoutTests/http/tests/contentextensions/character-set-basic-support.html.json.
* LayoutTests/http/tests/contentextensions/crash-xhr.json: Renamed from LayoutTests/http/tests/contentextensions/crash-xhr.html.json.
* LayoutTests/http/tests/contentextensions/cross-origin-content-rules.json: Renamed from LayoutTests/http/tests/contentextensions/cross-origin-content-rules.html.json.
* LayoutTests/http/tests/contentextensions/css-display-none-after-ignore-previous-rules.json: Renamed from LayoutTests/http/tests/contentextensions/css-display-none-after-ignore-previous-rules.html.json.
* LayoutTests/http/tests/contentextensions/css-display-none-font-shadow-tree.json: Renamed from LayoutTests/http/tests/contentextensions/css-display-none-font-shadow-tree.html.json.
* LayoutTests/http/tests/contentextensions/css-display-none-keyframe.json: Renamed from LayoutTests/http/tests/contentextensions/css-display-none-keyframe.html.json.
* LayoutTests/http/tests/contentextensions/css-display-none-overflows-rule-data-1.json: Renamed from LayoutTests/http/tests/contentextensions/css-display-none-overflows-rule-data-1.html.json.
* LayoutTests/http/tests/contentextensions/css-display-none-overflows-rule-data-2.json: Renamed from LayoutTests/http/tests/contentextensions/css-display-none-overflows-rule-data-2.html.json.
* LayoutTests/http/tests/contentextensions/css-display-none-overflows-rule-data-3.json: Renamed from LayoutTests/http/tests/contentextensions/css-display-none-overflows-rule-data-3.html.json.
* LayoutTests/http/tests/contentextensions/css-display-none-overflows-rule-data-4.json: Renamed from LayoutTests/http/tests/contentextensions/css-display-none-overflows-rule-data-4.html.json.
* LayoutTests/http/tests/contentextensions/css-display-none-overflows-rule-data-5.json: Renamed from LayoutTests/http/tests/contentextensions/css-display-none-overflows-rule-data-5.html.json.
* LayoutTests/http/tests/contentextensions/css-display-none-overflows-rule-data-6.json: Renamed from LayoutTests/http/tests/contentextensions/css-display-none-overflows-rule-data-6.html.json.
* LayoutTests/http/tests/contentextensions/css-display-none-with-different-case-sensitivity-are-not-merged.json: Renamed from LayoutTests/http/tests/contentextensions/css-display-none-with-different-case-sensitivity-are-not-merged.html.json.
* LayoutTests/http/tests/contentextensions/css-display-none-with-different-triggers-are-not-merged.json: Renamed from LayoutTests/http/tests/contentextensions/css-display-none-with-different-triggers-are-not-merged.html.json.
* LayoutTests/http/tests/contentextensions/css-display-none.json: Renamed from LayoutTests/http/tests/contentextensions/css-display-none.html.json.
* LayoutTests/http/tests/contentextensions/dns-prefetch-blocked.json: Renamed from LayoutTests/http/tests/contentextensions/dns-prefetch-blocked.html.json.
* LayoutTests/http/tests/contentextensions/domain-rules.json: Renamed from LayoutTests/http/tests/contentextensions/domain-rules.html.json.
* LayoutTests/http/tests/contentextensions/fetch-redirect-blocked.json: Renamed from LayoutTests/http/tests/contentextensions/fetch-redirect-blocked.html.json.
* LayoutTests/http/tests/contentextensions/filters-with-quantifiers-combined.json: Renamed from LayoutTests/http/tests/contentextensions/filters-with-quantifiers-combined.html.json.
* LayoutTests/http/tests/contentextensions/font-display-none-repeated-layout.json: Renamed from LayoutTests/http/tests/contentextensions/font-display-none-repeated-layout.html.json.
* LayoutTests/http/tests/contentextensions/frame-url.json: Renamed from LayoutTests/http/tests/contentextensions/frame-url.html.json.
* LayoutTests/http/tests/contentextensions/hide-on-csp-report.json: Renamed from LayoutTests/http/tests/contentextensions/hide-on-csp-report.py.json.
* LayoutTests/http/tests/contentextensions/hide-on-ping-with-ping-that-redirects.json: Renamed from LayoutTests/http/tests/contentextensions/hide-on-ping-with-ping-that-redirects.html.json.
* LayoutTests/http/tests/contentextensions/hide-on-ping.json: Renamed from LayoutTests/http/tests/contentextensions/hide-on-ping.html.json.
* LayoutTests/http/tests/contentextensions/invalid-selector.json: Renamed from LayoutTests/http/tests/contentextensions/invalid-selector.html.json.
* LayoutTests/http/tests/contentextensions/main-resource-redirect-blocked.json: Renamed from LayoutTests/http/tests/contentextensions/main-resource-redirect-blocked.py.json.
* LayoutTests/http/tests/contentextensions/main-resource-redirect-error.json: Renamed from LayoutTests/http/tests/contentextensions/main-resource-redirect-error.html.json.
* LayoutTests/http/tests/contentextensions/main-resource.json: Renamed from LayoutTests/http/tests/contentextensions/main-resource.html.json.
* LayoutTests/http/tests/contentextensions/make-https.json: Renamed from LayoutTests/http/tests/contentextensions/make-https.html.json.
* LayoutTests/http/tests/contentextensions/media-filtered.json: Renamed from LayoutTests/http/tests/contentextensions/media-filtered.html.json.
* LayoutTests/http/tests/contentextensions/popups.json: Renamed from LayoutTests/http/tests/contentextensions/popups.html.json.
* LayoutTests/http/tests/contentextensions/preconnect-blocked.json: Renamed from LayoutTests/http/tests/contentextensions/preconnect-blocked.html.json.
* LayoutTests/http/tests/contentextensions/prefetch-blocked.json: Renamed from LayoutTests/http/tests/contentextensions/prefetch-blocked.html.json.
* LayoutTests/http/tests/contentextensions/reload-without-contentextensions.json: Renamed from LayoutTests/http/tests/contentextensions/reload-without-contentextensions.html.json.
* LayoutTests/http/tests/contentextensions/root-actions.json: Renamed from LayoutTests/http/tests/contentextensions/root-actions.html.json.
* LayoutTests/http/tests/contentextensions/script-onerror.json: Renamed from LayoutTests/http/tests/contentextensions/script-onerror.html.json.
* LayoutTests/http/tests/contentextensions/service-worker-block-everything-if-localhost.json: Renamed from LayoutTests/http/tests/contentextensions/service-worker-block-everything-if-localhost.html.json.
* LayoutTests/http/tests/contentextensions/service-worker.https.json: Renamed from LayoutTests/http/tests/contentextensions/service-worker.https.html.json.
* LayoutTests/http/tests/contentextensions/single-css-display-none.json: Renamed from LayoutTests/http/tests/contentextensions/single-css-display-none.html.json.
* LayoutTests/http/tests/contentextensions/style-resolver-changed-reentrancy.json: Renamed from LayoutTests/http/tests/contentextensions/style-resolver-changed-reentrancy.html.json.
* LayoutTests/http/tests/contentextensions/subresource-redirect-blocked.json: Renamed from LayoutTests/http/tests/contentextensions/subresource-redirect-blocked.html.json.
* LayoutTests/http/tests/contentextensions/sync-xhr-blocked.json: Renamed from LayoutTests/http/tests/contentextensions/sync-xhr-blocked.html.json.
* LayoutTests/http/tests/contentextensions/sync-xhr-redirection-blocked.json: Renamed from LayoutTests/http/tests/contentextensions/sync-xhr-redirection-blocked.html.json.
* LayoutTests/http/tests/contentextensions/test-jump-table-bytecode-generation.json: Renamed from LayoutTests/http/tests/contentextensions/test-jump-table-bytecode-generation.html.json.
* LayoutTests/http/tests/contentextensions/text-track-blocked.json: Renamed from LayoutTests/http/tests/contentextensions/text-track-blocked.html.json.
* LayoutTests/http/tests/contentextensions/top-url.json: Renamed from LayoutTests/http/tests/contentextensions/top-url.html.json.
* LayoutTests/http/tests/contentextensions/two-distinguishable-css-display-none-rules-on-main-resource.json: Renamed from LayoutTests/http/tests/contentextensions/two-distinguishable-css-display-none-rules-on-main-resource.html.json.
* LayoutTests/http/tests/contentextensions/video-element-resource-type.json: Renamed from LayoutTests/http/tests/contentextensions/video-element-resource-type.html.json.
* LayoutTests/http/tests/inspector/network/contentextensions/blocked-websocket-crash.json: Renamed from LayoutTests/http/tests/inspector/network/contentextensions/blocked-websocket-crash.html.json.
* LayoutTests/http/tests/websocket/tests/hybi/contentextensions/block-cookies-worker.json: Copied from LayoutTests/http/tests/websocket/tests/hybi/contentextensions/block.html.json.
* LayoutTests/http/tests/websocket/tests/hybi/contentextensions/block-cookies-worker.py.json: Removed.
* LayoutTests/http/tests/websocket/tests/hybi/contentextensions/block-cookies.json: Copied from LayoutTests/http/tests/websocket/tests/hybi/contentextensions/block.html.json.
* LayoutTests/http/tests/websocket/tests/hybi/contentextensions/block-cookies.py.json: Removed.
* LayoutTests/http/tests/websocket/tests/hybi/contentextensions/block-worker.json: Renamed from LayoutTests/http/tests/websocket/tests/hybi/contentextensions/block-worker.html.json.
* LayoutTests/http/tests/websocket/tests/hybi/contentextensions/block.json: Renamed from LayoutTests/http/tests/websocket/tests/hybi/contentextensions/block.html.json.
* LayoutTests/http/tests/websocket/tests/hybi/contentextensions/display-none-worker.json: Renamed from LayoutTests/http/tests/websocket/tests/hybi/contentextensions/display-none-worker.html.json.
* LayoutTests/http/tests/websocket/tests/hybi/contentextensions/display-none.json: Renamed from LayoutTests/http/tests/websocket/tests/hybi/contentextensions/display-none.html.json.
* LayoutTests/http/tests/websocket/tests/hybi/contentextensions/upgrade-worker.json: Renamed from LayoutTests/http/tests/websocket/tests/hybi/contentextensions/upgrade-worker.html.json.
* LayoutTests/http/tests/websocket/tests/hybi/contentextensions/upgrade.json: Renamed from LayoutTests/http/tests/websocket/tests/hybi/contentextensions/upgrade.html.json.
* LayoutTests/http/wpt/beacon/contentextensions/beacon-blocked.json: Renamed from LayoutTests/http/wpt/beacon/contentextensions/beacon-blocked.html.json.
* LayoutTests/http/wpt/beacon/contentextensions/beacon-redirect-blocked.json: Renamed from LayoutTests/http/wpt/beacon/contentextensions/beacon-redirect-blocked.html.json.
* LayoutTests/platform/glib/TestExpectations:
* Tools/WebKitTestRunner/TestController.cpp:
* Tools/WebKitTestRunner/mac/TestControllerMac.mm:
(WTR::TestController::configureContentExtensionForTest):

Canonical link: https://commits.webkit.org/304625@main
